Cargo.toml

```diff
@@ -11,10 +15,10 @@ crate-type = ["cdylib", "rlib", "staticlib"]
 
 [dependencies]
 nix = "0.24.1"
-regex = "*"
+regex = "1.6.*"
```

Comment:
Wildcard syntax would [exclude](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html) `regex` `1.7` or higher from being used. Is there a good reason for that? Seems like an unnecessary restriction, given that it should be semver compatible. Consider just using `1.6, which is shorter, more commonly used, and less restrictive?

Reply:
It makes sense to me.